### PR TITLE
Template Implementation and CRUD Project

### PR DESCRIPTION
**Pull request by afauzansw**
implement template from mazer into blade templating laravel and create controller for CRUD project

### CODE REVIEW

File: app/Http/Controllers/Project/ProjectController.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+     * Display a listing of the project.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Project::query()
+            ->with('user')
+            ->get();
+        return view('dashbord');
```
Comment (4RSIM3R) [inline app/Http/Controllers/Project/ProjectController.php:36]: dashboard or dashbord?

File: app/Http/Controllers/Project/ProjectController.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    /**
+     * Show the form for creating a new project.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
```
Comment (4RSIM3R) [inline app/Http/Controllers/Project/ProjectController.php:44]: delete it if unused

File: app/Http/Controllers/Project/ProjectController.php [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        return view('');
+    }
+
+    /**
+     * Remove the specified project from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
```
Comment (4RSIM3R) [inline app/Http/Controllers/Project/ProjectController.php:114]: Please add a delete function that can also delete an `exception` field that linked to project

**Review by 4RSIM3R — COMMENTED** [review]
Improve dulu QAQA

File: app/Http/Controllers/Project/ProjectController.php (hunk 2 above)
Comment (afauzansw) [reply app/Http/Controllers/Project/ProjectController.php:44]: Blm komplit semua,

File: app/Http/Controllers/Project/ProjectController.php (hunk 3 above)
Comment (afauzansw) [reply app/Http/Controllers/Project/ProjectController.php:114]: well

File: app/Http/Controllers/Project/ProjectController.php (hunk 1 above)
Comment (afauzansw) [reply app/Http/Controllers/Project/ProjectController.php:36]: dashboard, has been replaced

**Pull request merged**